modules/aws/lb/README.md

```diff
@@ -0,0 +1,111 @@
+NLB
```

Comment:
Still missing the rest of the HTML and markdown from the template file here @Jakeasaurus